### PR TITLE
 query multiple tables

### PR DESCRIPTION
**Pull request by melkamu12**
### Vet Clinic Project:

In this PR, I added I create table Animals.

 ## **Activities done**
 * [ ] Create a table named owners with attributes of `id: integer, full_name: string and age: integer`
 * [ ] Create a table named species with attributes of `id: integer, name: string`
 * [ ] Modify animals table with drop column of `species and add column `species_id, owner_id`
 * [ ] Insert data into the owners and species tables and update animales table 

 ## **screenshot** 

 ## **Checklist**:
 * [ ] Follows the style guidelines of this project
 * [ ] follow gitflow

### CODE REVIEW

File: Schema.sql [hunk 1]
```diff
@@ -7,3 +7,18 @@ CREATE TABLE animals (
...
+);	
+CREATE TABLE species (
+	id SERIAL PRIMARY KEY,
+	name VARCHAR(100)
+);
+
+ALTER TABLE animals
+    DROP COLUMN species,
+    ADD COLUMN species_id INT,
+    ADD COLUMN owner_id INT;
```
Comment (jerryowusu) [inline Schema.sql]: Good job so far. However, kindly note the following and make changes accordingly: 

- [ ] The column species_id is a foreign key hence should reference the species table
- [ ]  The column owner_id is also a foreign key hence should reference the owners table. 
You can consider doing something like: 

```sql
ALTER TABLE animals ADD species_id int REFERENCES species(id);
ALTER TABLE animals ADD owner_id int REFERENCES owners(id);
```

File: data.sql [hunk 2]
```diff
@@ -12,4 +12,27 @@ VALUES
...
+    ('Ditto', '2022-05-14', 4, TRUE, 22);
+
+INSERT INTO owners (full_name, age)
+VALUES
+    ('Sam Smith', 34),
+	('Jennifer Orwell', 34),
+	('Bob', 34),
+	('Melody Pond', 34),
+	('Dean Winchester', 34),
+	('Jodie Whittaker', 34);
```
Comment (jerryowusu) [inline data.sql]: Kindly note that you need to insert the correct data in order to achieve the best optimum results. Most of the age values are wrong except that of `Sam Smith`. Kindly check the image below and insert the correct age values 

Screenshot from 2023-09-01 11-01-06

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @melkamu12 , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

so close


## Highlights
Used gitflow :+1: 
Nice branch name :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @melkamu12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Great job adding the necessary foreign keys and updating your database with the correct data  👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**